src/main/java/gov/usgs/wma/waterdata/LoadDiscreteGroundWater.java

```diff
@@ -31,6 +31,7 @@ public  ResultObject apply(RequestObject request) {
 		return processRequest(request);
 	}
 
+	@Transactional
```

Comment:
This annotation is also on the queries, and on the loader method below in this file.